pkg/gql/query/blocks.go

```diff
@@ -69,15 +68,7 @@ func newQueryBlock(b *block.Block) queryBlock {
 	feesPaid := uint64(0)
 
 	for _, tx := range b.Txs {
-		_, fee := tx.Values()
-		feesPaid += fee
-
-		if tx.Type() == core.Distribute {
-			// TODO: this should align with the actual distribute tx structure
-			// once that is finalized.
-			r := binary.LittleEndian.Uint64(tx.StandardTx().CallData)
-			reward += r
-		}
+		feesPaid += tx.Fee()
```

Comment:
This is not completely true. At the moment of writing, we are using GasLimit as indicator, later on we will' introduce GasSpent. The feesPaid should reflect the latter one

Reply:
Having in mind that `ExecutedTransaction` does not return `Fee`, feesPaid here will be always 0.